packit_service/worker/handlers/abstract.py

```diff
@@ -294,6 +294,15 @@ def __init__(
         self._project: Optional[GitProject] = None
         self._clean_workplace()
 
+    def get_package_name(self) -> Optional[str]:
```

Comment:
I'm not sure I follow here, what is this method needed for as for monorepos it returns None?

Reply:
ah okay, I get it now, thanks! Could you maybe add a comment about this to the docstring?